src/tests/example.spec.js

```diff
@@ -158,4 +158,79 @@ test.describe('Saucedemo app basic tests', () => {
             expect(cartPrice).toBe(selectedProducts[i].price);
         }
     });
+
+    test('check info about several added random products and Total Price on the "Checkout: Overview" page', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        // get all the elements of the products
+        const products = await app.inventory.product;
+        const productCount = await products.count();
+
+        // generate three unique indexes
+        const randomIndexes = getUniqueIndexes(productCount);
+        // const randomIndexes = new Set();
+        // const uniqueIndexesCount = Math.min(3, productCount);
+        // while (randomIndexes.size < uniqueIndexesCount) {
+        //     const randomIndex = Math.floor(Math.random() * productCount);
+        //     randomIndexes.add(randomIndex);
+        // }
+
+        // store information about selected products
+        const selectedProducts = await app.inventory.addedItemToCartInfoByIndex(randomIndexes);
+
+        // go to the shopping cart
+        await app.inventory.shoppingCartClick();
+
+        // Checkout: Add Information
+        await app.shoppingCart.checkoutButtonClick();
```

Comment:
Let's talk about namings. Our methods should start with action-word to make the action clear. 
Like `openCheckout` or `proceedToCheckout` instead of `checkoutButtonClick`;
Like `openShoppingCart` instead of `shoppingCartClick`
Like `getAddedProductsInfo` or `getSelectedProductsInfo` instead of `addedItemToCartInfoByIndex`
And so on. I know that it is difficult to give it an appropriate name, but practice makes everything better